index.html

```diff
@@ -16,14 +16,20 @@
   <body>
     <header>
       <nav>
-        <a href="#home"> <img src="images/My-Logo.png" /> </a>
+        <div id="logo">Pray Eddy</div> 
```

Comment:
**The lines that trigger these behaviours were not changed in this PR**

- [ ] **Required:**
- **Consider this gif**

![mobile_men](https://user-images.githubusercontent.com/27709832/202180972-a1785a50-d4f8-4016-b935-6011f7ea4105.gif)

- Kindly ensure that the page fits into the screens.
- I observed that the nav lists when clicked on do not scroll the page to that section, kindly fix this.